### ISSUE TITLE
ReferenceCatalog ListVersions returns 500 when using SortDirection

### ISSUE THREAD

**Issue by dennis-kuypers-gcx** [stoiveyp/Alexa.NET.Management#77]
Using Reference Catalog ListVersions always returns 500 error codes when I try to use a SortDirection. 

In the parameter documentation table they omitted a parameter that is present in the template url
`GET /v1/skills/api/custom/interactionModel/catalogs/{catalogId}/versions?maxResults={maxResults}&nextToken={nextToken}&sortDirection={sortDirection}&sortField={sortField}`

The 500 could be due to the missing `sortField`. However I have not been able to make this work using the ask-cli.
This issue is probably not actionable until we know how to use the API properly.

I'll update this ticket when I know more.

_Then we can probably also choose proper overloads that match the api (for example it looks like the nextToken contains the count, so maxResults can maybe be omitted when using next tokens)_

**Comment by stoiveyp** [comment]
Yeah I wasn't sure if sortField should be there or not - so thanks for raising the impact to keep track of it.

When my working day has finished I'll raise an issue in the developer forums to hopefully get some clarity 🤞

**Comment by dennis-kuypers-gcx** [comment]
I also raised a support ticket - but a forum entry won't hurt as we can also share results there 👍

**Comment by dennis-kuypers-gcx** [comment]
Quick Update: Alexa Developer Support responded that they are working on a fix - however they can not provide an ETA. I asked them to tell me once the fix was rolled out so that we can continue here.
